json_logic/__init__.py

```diff
@@ -1,201 +1,457 @@
+    '!!': _truthy,
+    '!': _falsy,
+    'log': _log,
+    'in': _in,
+    'cat': _concatenate,
+    '+': _add,
+    '-': _subtract,
+    '*': _multiply,
+    '/': _divide,
+    '%': _modulo,
+    'min': _minimal,
+    'max': _maximal,
+    'merge': _merge,
+}
+
+
+# Unsupported operations
+
+def _count(*args):
+    """Execute 'count' operation unsupported by core JsonLogic."""
+    return sum(1 if a else 0 for a in args)
+
+
+unsupported_operations = {
+    'count': _count
+}
+
+
+# Logical operations
+
+def _if(data, *args):
+    """
+    Evaluate chainable conditions with multiple 'else if' support and return
+    the corresponding evaluated argument based on the following patterns:
+
+    if 0 then 1 else None
+    if 0 then 1 else 2
+    if 0 then 1 else if 2 then 3 else 4
+    if 0 then 1 else if 2 then 3 else if 4 then 5 else 6
+
+    - If no arguments are given then return None.
+    - If only one argument is given the evaluate and return it.
+    - If two arguments are given then evaluate the first one, if it evaluates
+      to True return evaluated second argument, otherwise return None.
+    - If three arguments are given then evaluate the first one, if it evaluates
+      to True return evaluated second argument, otherwise return evaluated
+      third argument.
+    - For more then 3 arguments:
+        - If the first argument evaluates to True then evaluate and return
+          the second argument.
+        - If the first argument evaluates to False then jump to the next pair
+          (e.g.: from 0,1 to 2,3) and evaluate them.
+    """
+    for i in range(0, len(args) - 1, 2):
+        if _truthy(jsonLogic(args[i], data)):
+            return jsonLogic(args[i + 1], data)
+    if len(args) % 2:
+        return jsonLogic(args[-1], data)
+    else:
+        return None
+
+
+def _and(data, *args):
+    """
+    Evaluate and logically join arguments using the 'and' operator.
+    If all arguments evaluate to True return the last (truthy) one (meaning
+    that the whole expression evaluates to True).
+    Otherwise return first countered falsy argument (meaning that the whole
+    expression evaluates to False).
+    """
+    current = False
+    for current in args:
+        current = jsonLogic(current, data)
+        if _falsy(current):
+            return current  # First falsy argument
+    return current  # Last argument
+
+
+def _or(data, *args):
+    """
+    Evaluate and logically join arguments using the 'or' operator.
+    If at least one argument evaluates to True - return it
+    (meaning that the whole expression evaluates to True).
+    Otherwise return the last (falsy) argument (meaning that the whole
+    expression evaluates to False).
+    """
+    current = False
+    for current in args:
+        current = jsonLogic(current, data)
+        if _truthy(current):
+            return current  # First truthy argument
+    return current  # Last argument
+
+
+logical_operations = {
+    'if': _if,
```

Comment:
You need to add
  "?:": _if,
here too and remove it from the UNSUPPORTED_OPERATIONS in the tests.py file.